vector/v.info/parse.c

```diff
@@ -87,3 +87,36 @@ void parse_args(int argc, char **argv, char **input, char **field, int *history,
         }
     }
 }
+
+void parse_history_json(char *buf, char *command, char *gisdbase,
+                        char *location, char *mapset, char *user, char *date,
+                        char *mapset_path, JSON_Array *record_array)
```

Comment:
The function ended up here just because of its name. The parsing here is parsing command line input. I don't

In the tool itself, the function is more linked to the print anyway, so just keep it there for simplicity. Actually, in the current form, it is also very much linked to that print code.

In any case, please document split the function into two and document them with a sentence or two. It took me some time to orient myself in this code although it is actually straightforward and nicely written otherwise.